css/style.css

```diff
@@ -2,103 +2,222 @@ html {
     background: lightgrey;
 }
 
-#blueprint {
-    position: absolute;
-    display: none;
-    top: 50px;
-    left: 220px;
-    height: 300px;
-    width: 400px;
-    border: 2px solid black;
-    z-index: 9999;
-    opacity: 1;
-    text-align: center;
-    background: black;
+.factorio__container {

```

Comment:
Are we going to be using BEM style css classes?

Reply:
Didn't really think about it when writing the css, what do you think?

Reply:
For now, it's ok, but i'll open a ticket to let us discuss what convention to use.

Reply:
Actually, doing some research on BEM, it seems to be a pretty standard convention for CSS. Since we want to keep our dependencies down for this project, SASS is not exactly an option.

BEM should work well for the project, i just personally dislike the number of underscores one needs to type 😜

Reply:
Yeah I understand, if we decide on something else it won't be that difficult to change.
I personally think it would be nice with some precompilers, but I agree that it would increase the threshold for admitting code.